cirq/protocols/json_serialization.py

```diff
@@ -126,7 +126,11 @@ def two_qubit_matrix_gate(matrix):
                 cirq.experiments.SingleQubitReadoutCalibrationResult,
                 'SwapPowGate': cirq.SwapPowGate,
                 'SycamoreGate': cirq.google.SycamoreGate,
+<<<<<<< HEAD
```

Comment:
Remove these conflicts

Reply:
Hi. Thanks for the feedback. What do you mean ? Isn't line 112 necessary for serialization?

Reply:
Line 129 says 

`<<<<<<< HEAD`

This is not standard Python source code. It's actually from git telling you that you have conflicts.

Reply:
ok, right. Thanks.